test/testfixture/make_functions.go

```diff
@@ -137,7 +137,7 @@ func makeIterations(fxt *TestFixture) error {
 		fxt.Iterations[i] = &iteration.Iteration{
 			Name:        testsupport.CreateRandomValidTestName("iteration "),
 			Description: &desc,
-			UserActive:  &f,
+			UserActive:  f,
```

Comment:
When you remove the pointer you can remove the `f := false` above and this line all 
together.

Reply:
@kwk removed it here 0f50de0